examples/src/main/java/org/apache/spark/examples/mllib/JavaFPGrowthExample.java

```diff
@@ -25,32 +25,49 @@
 import org.apache.spark.SparkConf;
 import org.apache.spark.api.java.JavaRDD;
 import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.api.java.function.Function;
 import org.apache.spark.mllib.fpm.FPGrowth;
 import org.apache.spark.mllib.fpm.FPGrowthModel;
 
 /**
  * Java example for mining frequent itemsets using FP-growth.
+ * Example usage:  ./bin/run-example org.apache.spark.examples.mllib.JavaFPGrowthExample
```

Comment:
`run-example` doesn't need `org.apache.spark.examples.` So this could be a one-liner:

~~~
.bin/run-example mllib.JavaFPGrowthExample ./data/mllib/sample_fpgrowth.txt
~~~